runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsPlugin.java

```diff
@@ -241,45 +238,63 @@ public void onConfigChanged(ConfigChanged configChanged)
 	@Subscribe
 	public void onFocusChanged(FocusChanged focusChanged)
 	{
-		// If you somehow manage to right click while holding shift, then click off screen
```

Comment:
This comment probably shouldn't be removed.

Reply:
It doesn't apply any more due to the right click menu closing when the client loses focus. It's also now very obvious what's happening.